src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Inject {
```

Comment:
The `@Inject` annotation is correctly defined with the `@Retention` policy set to `RUNTIME`. This is necessary for runtime reflection. No changes needed.